# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gtk-vnc/sasl2.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur gtk-vnc-0.3.10-orig/src/gvnc.c gtk-vnc-0.3.10/src/gvnc.c
--- gtk-vnc-0.3.10-orig/src/gvnc.c	2010-01-15 22:21:28.000000000 +0000
+++ gtk-vnc-0.3.10/src/gvnc.c	2010-01-15 22:22:44.000000000 +0000
@@ -47,7 +47,7 @@
 #include <gnutls/x509.h>
 
 #if HAVE_SASL
-#include <sasl/sasl.h>
+#include <sasl2/sasl.h>
 #endif
 
 #include <zlib.h>
